packages/stack.rb

```diff
@@ -3,13 +3,10 @@
 class Stack < Package
   description 'The Haskell Tool Stack - Stack is a cross-platform program for developing Haskell projects. It is aimed at Haskellers both new and experienced.'
   homepage 'https://docs.haskellstack.org/en/stable/README/'
-  version '2.1.3'
+  version '2.1.3-1'
 
   case ARCH
-  when 'aarch64'
-    source_url 'https://github.com/commercialhaskell/stack/releases/download/v2.1.3/stack-2.1.3-linux-aarch64.tar.gz'
-    source_sha256 '96394b616e29b3bd2479a3712e2c0c375a7cfd748b613f7c6c3186d7ae92977b'
-  when 'armv7l'
+  when 'aarch64', 'armv7l'
```

Comment:
The stack tool itself runs on ARM but the tools which make up the stack are still unavailable for aarch64 and possibly ARMv7l (haven't tried on a machine yet or read their issue queue in a while). We should probably just make it unavailable for aarch64 for now.